cli/admin/src/main/java/io/pravega/cli/admin/debug/ReaderGroupStreamFileParsingCommand.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.admin.debug;
+
+import io.pravega.cli.admin.AdminCommand;
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.client.admin.impl.ReaderGroupManagerImpl;
+import io.pravega.client.state.impl.UpdateOrInitSerializer;
+import lombok.Cleanup;
+import lombok.val;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.nio.ByteBuffer;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * The command takes input stream file and output stream file as argument.
+ * It parses the serialized ReaderGroup Stream file to a human readable file.
+ */
+public class ReaderGroupStreamFileParsingCommand extends AdminCommand {
+    private final static int HEADER = 4;
+    private final static int LENGTH = 4;
+    private final static int TYPE = 0;
+
+    public ReaderGroupStreamFileParsingCommand(CommandArgs args) {
+        super(args);
+    }
+    
+    @Override
+    public void execute() {
+        try {
+            ensureArgCount(2);
+            String inputFileName = getInputStreamFilename(getCommandArgs().getArgs());
+            String outputFileName = getOutputStreamFilename(getCommandArgs().getArgs());
```

Comment:
This should be rewritten to make use of either RevisionedStreamClient/ByteStreamClient or the Admin Gateway (raw read of the segment file). 

How is this file supposed to get to your local filesystem in the first place? How do you download it from Pravega?